README.md

```diff
@@ -162,6 +162,20 @@ The following work on normal and visual modes:
 
 -   `:Tocv`: Same as `:Toc` for symmetry with `:Toch` and `Tocv`.
 
+## Issue reporting
+
+Issues are tracked within GitHub.
+
+When reporting issues, your report is more effective if you include a minimal example file that reproduces the problem. Try to trim out as much as possible, until you have the smallest possible file that still reproduces the issue. Paste the example inline into your issue report, quoted using four spaces at the beginning of each line, like this example from issue [#189](https://github.com/plasticboy/vim-markdown/issues/189):
+
+```
+Minimal example:
+
+    ```
+    =
+    ```
+    bad!
+```
```

Comment:
OK, if you think this really helps, let's merge it. Can you move this section into CONTRIBUTING.md ? It's the standard GitHub location for this kind of thing.